.gitignore

```diff
@@ -36,10 +36,11 @@
 /doc/doxygen/xml
 /doc/sphinx/_build
 .vscode
+venv
```

Comment:
remove? we usually do not put in the master things related to your local set up ....